src/cli.ts

```diff
@@ -51,26 +51,27 @@ export function printUsage() {
    zx [options] <script>
 
  ${chalk.bold('Options')}
-   --quiet              suppress any outputs
-   --verbose            enable verbose mode
-   --shell=<path>       custom shell binary
-   --prefix=<command>   prefix all commands
-   --postfix=<command>  postfix all commands
-   --cwd=<path>         set current directory
-   --eval=<js>, -e      evaluate script
-   --ext=<.mjs>         default extension
-   --install, -i        install dependencies
-   --version, -v        print current zx version
-   --help, -h           print help
-   --repl               start repl
-   --experimental       enables experimental features (deprecated)
+   --quiet                          suppress any outputs
+   --verbose                        enable verbose mode
+   --shell=<path>                   custom shell binary
+   --prefix=<command>               prefix all commands
+   --postfix=<command>              postfix all commands
+   --cwd=<path>                     set current directory
+   --eval=<js>, -e                  evaluate script
+   --ext=<.mjs>                     default extension
+   --install, -i                    install dependencies
+   --registry=<npm registry URL>    custom npm registry URL dependencies works with install flag
```

Comment:
I'd suggest:
`--registry=<URL>    npm registry, defaults to https://registry.npmjs.org/`

Update also `man/zx.1`

Reply:
Cool suggest!

I update  `man/zx.1` and description for cli